[Quazip] mac compilation error do not find Qt5

On mac compilation i've got:

```
CMake Error at /usr/local/Cellar/cmake/3.18.4/share/cmake/Modules/FindQt4.cmake:1314 (message):
  Found unsuitable Qt version "" from NOTFOUND, this code requires Qt 4.x
Call Stack (most recent call first):
  CMakeLists.txt:34 (find_package)


-- Configuring incomplete, errors occurred!
See also "/System/Volumes/Data/builds/workspace/music-macos-free/build/ExtProjs/quazip/CMakeFiles/CMakeOutput.log".
```

When i do a ccmake on medInria-public build i can see that Qt5 is found. However, in the quazip build directory, i've got:
 * BUILD_WITH_QT4 at OFF
 * but Qt5Core_DIR at Qt5Core_DIR-NOTFOUND
 * and QT_QMAKE_EXECUTABLE to NOTFOUND

--- a/superbuild/projects_modules/quazip.cmake
+++ b/superbuild/projects_modules/quazip.cmake
@@ -46,6 +46,8 @@ set(cmake_args
   -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>
   # to find zlib 
   -DCMAKE_PREFIX_PATH:FILEPATH=${zlib_DIR}
+  # to find Qt5
+  -DQt5_DIR=${Qt5_DIR}
 )
 
 ## #############################################################################

